integration_test.go

```diff
@@ -477,3 +648,44 @@ func getContractAddressesFromContractRegistry(ethHttpUrl string) ContractAddress
 		Erc20MockStrategy:      erc20MockStrategyAddr,
 	}
 }
+
+func NewEthHttpClient(rpcAddress string) (*ethclient.Client, error) {
+	client, err := ethclient.Dial(rpcAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	return client, nil
+}
+
+func createWalletForOperator(t *testing.T, privKeyHex string, ethClient *ethclient.Client) walletsdk.Wallet {
+	logger, err := logging.NewZapLogger(logging.Development)
```

Comment:
@shrimalmadhur didn't we deprecate zaplogger? We prob should otherwise, to force people to use slog everywhere

Reply:
I don't think we deprecated it. But we replaced everything with Slog everywhere. Can deprecate it in future version since slog is the way to go!

Reply:
I'll replace with slog

Reply:
https://github.com/Layr-Labs/avs-sync/pull/45/commits/080e4cbe04e753bdaffe44d7cb680d270807bb6c